src/App.jsx

```diff
@@ -3,26 +3,58 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getUsers, getTodos } from './api/api';
 
 class App extends React.Component {
   state = {
-    todos: [],
+    todos: null,
     selectedUserId: 0,
+    users: [],
   };
 
+  async componentDidMount() {
+    const [todos, users] = await Promise.all([
+      getTodos(),
+      getUsers(),
+    ]);
+
+    this.setState({
+      todos,
+      users,
+    });
+  }
+
+  selectedUser = (id) => {
```

Comment:
```suggestion
  selectUser = (id) => {
```
функция доджна называться так, чтобы было понятно что она делает(выбирает пользователя? selectUser).

Reply:
selectedUser - это выбранный пользователь